service-tests/github.js

```diff
@@ -447,3 +447,31 @@ t.create('github pull request check state')
 t.create('github pull request check contexts')
   .get('/status/contexts/pulls/badges/shields/1110.json')
   .expectJSONTypes(Joi.object().keys({ name: 'checks', value: '1 failure' }));
+
+t.create('top language')
+.get('/languages/top/badges/shields.json')
+.expectJSONTypes(Joi.object().keys({
+  name: Joi.equal('JavaScript'),
+  value: Joi.string().regex(/^([1-9]?[0-9]\.[0-9]|100\.0)%$/),
+}));
+
+t.create('top language with empty repository')
+.get('/languages/top/pyvesb/emptyrepo.json')
+.expectJSONTypes(Joi.object().keys({
+  name: Joi.equal('language'),
+  value: Joi.equal('none'),
+}));
+
+t.create('language count')
+.get('/languages/count/badges/shields.json')
+.expectJSONTypes(Joi.object().keys({
+  name: Joi.equal('languages'),
+  value: Joi.string().regex(/^[0-9]*$/),
+}));
+
+t.create('code size in bytes for all languages')
+.get('/languages/bytes/badges/shields.json')
+.expectJSONTypes(Joi.object().keys({
+  name: Joi.equal('code size'),
+  value: Joi.string().regex(/^[0-9]*(k|M|G|T|P|E|Z|Y)B$/),
```

Comment:
You can use the `isFileSize` helper from `service-tests/helpers/validators.js`.

Reply:
My bad, I had forked the repository before the helpers were added, this is now fixed.